### PR TITLE
clarified doc

### PR DESCRIPTION
**Pull request by jcohenadad**

### CODE REVIEW

**Review by kousu — CHANGES_REQUESTED** [review]
I think this explanation is too specific to the specific issue you had. It reads like a note-to-self not a note-to-the-lab.

It also doesn't actually work! 'true' does almost the opposite of what we want: it normalizes on CRLF line-endings for everyone (so ^M everywhere instead of nowhere).

https://stackoverflow.com/a/4683783/2898673 and its follow up https://stackoverflow.com/a/34089670/2898673 are correct and more reliable. I'll patch them in on this branch, if you don't mind.

**Comment by jcohenadad** [comment]
> I think this explanation is too specific to the specific issue you had. It reads like a note-to-self not a note-to-the-lab.
> 
> It also doesn't actually work! 'true' does almost the opposite of what we want: it normalizes on CRLF line-endings for everyone (so ^M everywhere instead of nowhere).
> 
> https://stackoverflow.com/a/4683783/2898673 and its follow up https://stackoverflow.com/a/34089670/2898673 are correct and more reliable. I'll patch them in on this branch, if you don't mind.

yes, that makes perfect sense!  thank you

**Comment by kousu** [comment]
https://adaptivepatchwork.com/2012/03/01/mind-the-end-of-your-line/

what a mess

tl;dr:

* `git config --global core.autocrlf false` doesn't do anything; and that's the point, you might set it as an optimization if you never have to deal with cross-platform work
* `git config --global core.autocrlf true` *forces* CRLF on your machine; it is meant to be set on Windows computers that need to do cross-platform work with Unix ones
* `git config --global core.autocrlf input` is almost correct; it might even work for us, but it's deprecated; it is used
* `echo "*  text=auto" >> .gitattributes` is the current recommended system: it means the internal copy in the repo gets LF, Unix contributors get LF and Windows contributors get CRLF; but unlike the previous methods, this is per-repo, not per-machine
* `echo "*  text=auto eol=crlf" >> .gitattributes` should(?) be the same as `git config --global core.autocrlf true`, but again, it's coded into the repo, not into each workstation.

**Comment by kousu** [comment]
and this answer is messily written but it has the best summary of the conclusion I came to: https://stackoverflow.com/a/11199598/2898673

**Review by kousu — APPROVED** [review]

**Comment by kousu** [comment]
I'll go through all the repos on `data.neuro` and also in https://github.com/spine-generic/ and apply this change to each of them. This is the unfortunate bit about the new strategy: it is boilerplate that leaks into every repo.
